Index: src/pullrequest/pullRequestCommentContent.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pullrequest/pullRequestCommentContent.ts b/src/pullrequest/pullRequestCommentContent.ts
--- a/src/pullrequest/pullRequestCommentContent.ts	
+++ b/src/pullrequest/pullRequestCommentContent.ts	
@@ -68,8 +68,6 @@
 
     }
 
-    console.log("committerMap")
-    console.log(committerMap)
 
     let you = committersCount > 1 ? `you all` : `you`
     let lineOne = (input.getCustomNotSignedPrComment() || `<br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that $you sign our [Contributor License Agreement](${input.getPathToDocument()}) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>`).replace('$you', you)
